packages/main/src/components/protected/sidebar/components/Sidebar.jsx

```diff
@@ -216,11 +216,6 @@ const Sidebar = props => {
                 image={dmIcon}
                 link={`/workspace/${currentWorkspace}/video-chat`}
               />
-              <SingleRoom
-                name="Video Chat"
-                image={dmIcon}
-                link={`/workspace/${currentWorkspace}/video-chat`}
-              />
               <SingleRoom
                 name={`${t("workspace_chat.drafts")}`}
```

Comment:
What is t?

Reply:
Name t as translation or something better.

Reply:
Do you mean the video chat?

Reply:
the t you pulled from useTranslation, it's not descriptive.

Reply:
The t function somehow came as a property of the useTranslation object,

Reply:
It appeared as if I did it, because when I had to resolve the merge conflicts,  I did some copying and pasting